lr_face/data.py

```diff
@@ -299,7 +299,65 @@ def images(self) -> List[FaceImage]:
         for file in files:
             path = os.path.join(self.RESOURCE_FOLDER, file)
             identity = f'FORENFACE-{file[:3]}'
-            data.append(FaceImage(path, identity, source=str(self)))
+
+            # parse annotations in filename:
+            # cx -> camera number
+            # a = no hat, b = hat
+            annotation_code = os.path.splitext(file)[0][3:]
+            if annotation_code in ('lp', 'rp'):
+                yaw = Yaw.PROFILE
+                pitch = Pitch.FRONTAL
+                headgear = False
+                low_quality = False
+            elif annotation_code in ('lq', 'rq'):
+                yaw = Yaw.HALF_TURNED
+                pitch = Pitch.FRONTAL
+                headgear = False
+                low_quality = False
+            elif annotation_code in ('a', 'f', 'c1a7', 'c2a3', 'c3a8', 'c3a16', 'c4a7', 'c4a12', 'c5a3'):
```

Comment:
je zou de expressie nog in een dict kunnen stoppen ipv if/else, maar dat is niet een must have